scripts/apps/authoring/views/authoring-header.html

```diff
@@ -181,10 +183,11 @@
                  sd-width="{{editor.place.sdWidth}}"
                  ng-attr-title="{{schema.place.readonly === true ? readOnlyLabel : ''}}"
                  order="{{editor.place.order}}">
-                <label class="authoring-header__item-label">{{getLabelForFieldId('place')}}</label>
+                <label class="authoring-header__item-label" for="place">{{getLabelForFieldId('place')}}</label>
                 <div class="authoring-header__input-holder">
                     <div sd-meta-terms ng-if="!features.places_autocomplete && !single_value.locators"
                          class="data"
+                         id="place"
```

Comment:
It'd be good to use unique strings for identifiers in order to be able to find usages easily. If you grep for `place` or `title` there are too many matches and it's hard to see usages. We could add `a11y-` to IDs. Like `id="a11y-place"`.